common/src/main/java/org/apache/nemo/common/coder/IntEncoderFactory.java

```diff
@@ -59,7 +59,8 @@ public String toString() {
    * IntEncoder.
    */
   private final class IntEncoder implements Encoder<Integer> {
-
+    // Exclude from sonar-cloud check as DataOutputStream is Java standard lib
+    //NOSONAR
```

Comment:
Let's just remove the two comments here and let the other committers fix it :)

Reply:
I will fix it